spec/abstractops.html

```diff
@@ -1832,6 +1832,9 @@ <h1>
             1. Else if _Conversion_ is ~ToPositiveIntegerWithTruncation~, then
               1. Set _value_ to ? ToPositiveIntegerWithTruncation(_value_).
               1. Set _value_ to 𝔽(_value_).
+            1. Else if _Conversion_ is ~StringOnly~, then
+              1. If _value_ is an Object, set _value_ to ? ToString(_value_).
+              1. Else if _value_ is not a String, throw a *TypeError* exception.
```

Comment:
ToString will convert `null` to `"null"`, for example. Perhaps this might be more explicit?
```suggestion
              1. If _value_ is an Object, set _value_ to ? ToPrimitive(_value_, ~string~).
              1. If _value_ is not a String, throw a *TypeError* exception.
```

Reply:
No, that's not what i mean. I mean that `{ valueOf() { return null; } }` will ToString to a string, but with my suggestion, would throw.

Reply:
Ah, got it. Makes sense. Is there any downside of using ToPrimitive instead? If not then I'm inclined to follow your suggestion.

BTW, take a look at the latest spec text, it has the new AO that I had in mind.

Reply:
I don't see any downsides personally.

Reply:
Resolving because the latest iteration of this PR removes conversion of objects to strings.